core/domain/collection_validators_test.py

```diff
@@ -0,0 +1,1651 @@
+# coding: utf-8
+#
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for core.domain.collection_validators."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import ast
+import datetime
+import math
+import random
+import time
+import types
+
+from constants import constants
+from core.domain import collection_domain
+from core.domain import collection_services
+from core.domain import exp_domain
+from core.domain import exp_services
+from core.domain import prod_validation_jobs_one_off
+from core.domain import rights_domain
+from core.domain import rights_manager
+from core.domain import user_services
+from core.platform import models
+from core.tests import test_utils
+import feconf
+import python_utils
+import utils
+
+datastore_services = models.Registry.import_datastore_services()
+gae_search_services = models.Registry.import_search_services()
+
+USER_EMAIL = 'useremail@example.com'
+USER_NAME = 'username'
+CURRENT_DATETIME = datetime.datetime.utcnow()
+
+(
+    audit_models, collection_models,
```

Comment:
Is`audit_models` required ?

Reply:
Also, remove `email_models`, if not required!